dockerize-repository/docker-compose.yml

```diff
@@ -12,9 +12,7 @@ services:
             - "5432"
 
     b2share:
-        build:
-            context: ..
-            dockerfile: dockerize/Dockerfile
+        image: eudatb2share/b2share:2.0.1
```

Comment:
The `dockerize-repository` is what the users will clone and use as demo. The contents of this
folder will go as it is into the `EUDAT-B2SHARE/dockerize` repository (see first comment). For local testing we'll use `devenv`.